Makefile

```diff
@@ -183,6 +183,12 @@ test-in-docker:
 run-functional-tests: testnnp test-registry ecr-execution-role-image telemetry-test-image
 	. ./scripts/shared_env && go test -tags functional -timeout=30m -v ./agent/functional_tests/...
 
+build-image-for-ecr: netkitten volumes-test squid awscli image-cleanup-test-images fluentd taskmetadata-validator testnnp container-health-check-image telemetry-test-image ecr-execution-role-image-for-upload
+	@./scripts/setup-ecr
+
+ecr-execution-role-image-for-upload:
```

Comment:
Why do we need this?

Reply:
You mean ecr-execution-role-image-for-upload?. Seems we need this image and I do not want to modify original files in misc/ecr folder which not only build the image, but also upload to the account ecr. While I only need the logic to build the image.

Reply:
Are we able to standardize aws account specific configurations like this to a naming standard that can be A) inferred if not provided, and B) overwrite if need?